src/kaltura-player.js

```diff
@@ -76,19 +94,30 @@ class KalturaPlayer extends FakeEventTarget {
     const providerResult = this._provider.getMediaConfig(mediaInfo);
     providerResult
       .then(
-        mediaConfig => this.setMedia(mediaConfig),
+        (providerMediaConfig: ProviderMediaConfigObject) => {
+          const mediaConfig = Utils.Object.copyDeep(providerMediaConfig);
+          if (mediaOptions) {
+            mediaConfig.playback = mediaConfig.playback || {};
+            mediaConfig.sources = mediaConfig.sources || {};
+            const {startTime} = mediaOptions;
+            if (typeof startTime === 'number') {
```

Comment:
@dan-ziv I think it would make a problem cause it'll be saved for the next media as we have today with startTime.
I think the better option is to set startTime on sources instead of playback config, it'll be possible to pass it on `setMedia` as well for specific media.

Reply:
agree, this will require changes in playkit-js where startTime can come from source.
I would actually remove playkit-js startTime config from playback all together and move it to sources level and align it here in kaltura-player to merge it if it was given on playback config with deprecated warning.

Reply:
after discussing with @dan-ziv we should handle this in separate PR.
This PR solves issue with being able to pass `mediaOptions` in run time without requiring app to do reset, configure, loadMedia and just do loadMedia.
the startTime is indeed an issue to take care of, but as mentioned it will be done separately.

Reply:
OK, we need to add to this PR to remove the playback config from setMedia config as well.